### ISSUE TITLE
Is this Generator compatible with  chrome manifest v3 

### ISSUE THREAD

**Issue by anuraggautam77** [yeoman/generator-chrome-extension#243]
> https://developer.chrome.com/docs/extensions/mv3/intro/

**Comment by cleydyr** [comment]
I don't think so, as the manifest version is [hard-coded](https://github.com/yeoman/generator-chrome-extension/blob/main/app/chrome-manifest.js#L78).
